target/linux/ramips/dts/mt7621_zbtlink_zbt-wg1608-32m.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_zbtlink_zbt-wg1608.dtsi"
+
+/ {
+	compatible = "zbtlink,zbt-wg1608-32m", "zbtlink,zbt-wg1608-16m", "zbtlink,zbt-wg1608", "mediatek,mt7621-soc";
```

Comment:
`"zbtlink,zbt-wg1608-16m", "zbtlink,zbt-wg1608",` is redundant I think.

Reply:
I removed it. Thanks.